streaming/src/main/scala/org/apache/spark/streaming/receiver/ReceivedBlockHandler.scala

```diff
@@ -79,7 +93,7 @@ private[streaming] class BlockManagerBasedBlockHandler(
       throw new SparkException(
         s"Could not store $blockId to block manager with storage level $storageLevel")
     }
-    BlockManagerBasedStoreResult(blockId)
+    BlockManagerBasedStoreResult(blockId, numRecords)
```

Comment:
The bytebuffer is expected to have multiple serialized records in it, so I think it is more wrong to count it as 1. So None is better than Some(1)